referral/models.py

```diff
@@ -16,22 +18,23 @@ class Referral(Note):
 
     # Status if there are no referrals of a specific type
     # Used in aggregate_referral_status
-    NO_REFERRALS_CURRENTLY = "No referrals currently"
+    NO_REFERRALS_CURRENTLY = _("No referrals currently")
 
     REFERRAL_STATUSES = (
-        (STATUS_SUCCESSFUL, 'Successful'),
-        (STATUS_PENDING, 'Pending'),
-        (STATUS_UNSUCCESSFUL, 'Unsuccessful'),
+        (STATUS_SUCCESSFUL, _('Successful')),
+        (STATUS_PENDING, _('Pending')),
+        (STATUS_UNSUCCESSFUL, _('Unsuccessful')),
```

Comment:
Does this change the database? (I don't think so, I think that's what `STATUS_PENDING` does, but I'm not sure.)